theano/gpuarray/subtensor.py

```diff
@@ -7,6 +7,9 @@
 from six.moves import StringIO
 
 from theano import tensor, gof, Op
+from theano.gradient import grad_not_implemented
+import theano.tensor.clip
+import theano.tensor.minimum
```

Comment:
These lines are the source of the import errors.  There are no modules by those names.